styles.css

```diff
@@ -480,3 +503,183 @@ nav {
   background: #0d151e;
   border-radius: 10px;
 }
+
+/* --desktop-- */
+
+@media only screen and (min-width: 768px) {
+  .menu-image {
+    display: none;
+  }
+
+  .humbuger {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 32px;
+    margin-left: 70%;
+  }
+
+  .text_one {
+    background-size: contain;
+    background-image: url("/image/headerbg.svg");
+    width: 100%;
+    height: 600px;
+    padding-top: 5rem;
+    padding-left: 25%;
+    padding-right: 25%;
+  }
+
+  .noqs {
+    width: 100%;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .card-title {
+    width: 50%;
+    padding-top: 60px;
+    margin: 24px;
+  }
+
+  .cardimage {
+    max-width: 600px;
+    max-height: 220px;
+  }
+
+  .noqs.reversed {
+    flex-direction: row-reverse;
+  }
+
+  .about {
+    min-height: 1vh;
+    margin-top: 114px;
+    background-color: white;
+    border-top-right-radius: 40px;
+  }
+
+  .contain_noqs {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 1.5rem;
+    height: 100%;
+    padding-top: 114px;
+    padding-bottom: 113px;
+  }
+
+  .allabout {
+    display: flex;
+    flex-direction: row;
+    margin-left: 60px;
+    margin-right: 40px;
+    gap: 1rem;
+    width: 50%;
+  }
+
+  .language_list {
+    display: flex;
+    flex-direction: row;
+    width: 80px;
+    height: 80px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    gap: 10rem;
+    margin-left: 20px;
+  }
+
+  /* here */
+  .about_part {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 0.75rem;
+    background-size: 100%;
+    background-color: white;
+  }
+
+  .heading_about {
+    display: flex;
+    align-self: flex-start;
+    width: 100%;
+  }
+
+  .about_part p {
+    display: flex;
+    align-self: flex-start;
+    width: 100%;
+  }
+
+  .connect2 {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 1.2188rem;
+  }
+
+  .contact_sub {
+    background-color: #6070ff;
+    border-top-left-radius: 5rem;
+    margin: auto;
+    padding: 5rem;
+  }
+
+  .cont_act {
+    background-image: url('/image/contactdesk.png') no repeat;
+    background-color: #6070ff;
+    width: 100%;
+    background-size: contain;
+    border-top-left-radius: 80px;
+    padding-top: 30px;
+    margin: auto;
+    padding-bottom: 50px;
+    align-items: center;
+  }
+
+  .contact_items,
+  .form,
+  .contact_paragraph {
+    align-items: center;
+    width: 100%;
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+
+  .form_message {
+    resize: none;
+  }
+
+  .see-button {
```

Comment:
- [ ] Kindly make sure to use the proper styling for the buttons including the hover and active state, at hove make sure to remove the border radios and add styling for active state (when the user clicks on the button), this will make make your design more interactive and resemble to the provided design

|Hover state|Normale state|Figma buttons|
|-|-|-|
|![Screenshot_20220901_214729](https://user-images.githubusercontent.com/25227022/188010810-1474617d-121f-4496-bc9c-d617e0fb76fe.png)|![Screenshot_20220901_215511](https://user-images.githubusercontent.com/25227022/188011269-a3118b7a-3b47-4e2d-b23b-d131bca3f87f.png)|![Screenshot_20220901_215546](https://user-images.githubusercontent.com/25227022/188011314-b5898cc3-6417-45f1-ae4f-4da5f6beb2f7.png)|

Reply:
Good job following the advice of the previous reviewer